usaon_vta_survey/__init__.py

```diff
@@ -71,9 +71,11 @@ def before_request():
     from usaon_vta_survey.routes.users import users_bp
 
     app.register_blueprint(users_bp)
-    from usaon_vta_survey.routes.login import blueprint, login_bp
+    from usaon_vta_survey.routes.login import login_bp
 
     app.register_blueprint(login_bp)
+    from usaon_vta_survey.routes.google import blueprint
+
```

Comment:
I'm guessing the formatter was forcing these blank lines? What about grouping all the imports together, then all the registrations?  Or maybe we could just disable the formatting for these lines. No strong opinion or request to change anything here, just thinking out loud.

Reply:
went with grouping all the imports.